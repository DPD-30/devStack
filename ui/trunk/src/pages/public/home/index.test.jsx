import React from 'react'
import { renderToString } from 'react-dom/server'

import PagesHome from 'pages/public/home/index.jsx'

jest.unmock('pages/public/home/index.jsx')


describe('', () => {
    it('PagesHome should match snapshot', () => {
        const comp = renderToString(<PagesHome />)

        expect(comp).toMatchSnapshot()
    })
})
