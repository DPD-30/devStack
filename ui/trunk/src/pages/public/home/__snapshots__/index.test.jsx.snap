// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PagesHome should match snapshot 1`] = `
<div style="width:650px;margin:0 auto;text-align:center"
     class="jumbotron"
     data-reactroot
>
  <h3>
    Welcome!
  </h3>
  <br>
  <p>
    This application was created as a demonstration of technical capabilities.
  </p>
  <hr>
  <p>
    It leverages a modern and extensive technical stack, including:
  </p>
  <ul style="width:180px;text-align:left;margin:0 auto">
    <li>
      ReactJS
    </li>
    <li>
      Reflux
    </li>
    <li>
      OpenAPI
    </li>
    <li>
      C# WebAPI
    </li>
    <li>
      MS SQL Server
    </li>
    <li>
      MS Azure
    </li>
  </ul>
</div>
`;
